Otherwise, OroboROX is the default only if zero-install is available.

--- origsrc/ROX-Session/src/wm.c	2008-07-01 19:32:46.000000000 -0500
+++ src/ROX-Session/src/wm.c	2009-10-19 11:26:51.070841900 -0500
@@ -261,7 +261,7 @@ static void run_wm(const char *wm)
  */
 static const char *get_window_manager(void)
 {
-	const char *fallbacks[] = {"openbox", "xfwm4", "sawfish", "enlightenment",
+	const char *fallbacks[] = {"OroboROX", "openbox", "xfwm4", "sawfish", "enlightenment",
 				"wmaker", "icewm", "blackbox", "fluxbox", "metacity", "kwin",
 				"kwm", "fvwm2", "fvwm", "4Dwm", "twm"};
 
